nucypher/policy/policies.py

```diff
@@ -217,30 +221,51 @@ def __init__(self,
 
         self._threadpool = ThreadPool(minthreads=threadpool_size, maxthreads=threadpool_size, name=self._repr)
         self.log.info(f"NEM spinning up {self._threadpool}")
+        self._threadpool.callInThread(self._bail_on_timeout)
```

Comment:
Any reason we don't do this in start(...), where the other callInThread calls are done?

Reply:
Agree - I think this can be relocated to `Ursula.start`

Reply:
cc @jMyles

Reply:
:+1: